iOSClient/More/NCMore.swift

```diff
@@ -97,35 +111,43 @@ class NCMore: UIViewController, UITableViewDelegate, UITableViewDataSource {
         labelQuotaExternalSite.text = ""
         progressQuota.progressTintColor = NCBrandColor.shared.brandElement
 
-        // ITEM : Transfer
-        item = NKExternalSite()
-        item.name = "_transfers_"
-        item.icon = "arrow.left.arrow.right"
-        item.url = "segueTransfers"
-        item.order = 10
-        functionMenu.append(item)
+//        // ITEM : Transfer
+//        item = NKExternalSite()
+//        item.name = "_transfers_"
+//        item.icon = "arrow.left.arrow.right"
+//        item.url = "segueTransfers"
+//        item.order = 10
+//        functionMenu.append(item)
 
         // ITEM : Recent
         item = NKExternalSite()
         item.name = "_recent_"
-        item.icon = "clock.arrow.circlepath"
+        item.icon = "recent"
         item.url = "segueRecent"
         item.order = 20
         functionMenu.append(item)
 
-        // ITEM : Activity
+        // ITEM : Notification
         item = NKExternalSite()
-        item.name = "_activity_"
-        item.icon = "bolt"
-        item.url = "segueActivity"
-        item.order = 40
+        item.name = "_notifications_"
+        item.icon = "notification"
+        item.url = "segueNotification"
+        item.order = 30
         functionMenu.append(item)
 
+//        // ITEM : Activity
+//        item = NKExternalSite()
+//        item.name = "_activity_"
+//        item.icon = "bolt"
+//        item.url = "segueActivity"
+//        item.order = 40
+//        functionMenu.append(item)
```

Comment:
I guess only commenting functionMenu.append(item) will avoid showing Activity item instead of commenting whole code block.